src/Employer/Esfa.Recruit.Azure.Resources/azuredeploy.json

```diff
@@ -110,6 +110,10 @@
       "type": "bool",
       "defaultValue": false
     },
+	"WhitelistedProvidersList":{
```

Comment:
This needs to be added in the parameters file as well.

Reply:
ok..will do!

Reply:
Where is this even used? (rhetorical question)

Reply:
We provide a list of UKPRN's in the json file.